content/post/opera-ios.md

```diff
@@ -0,0 +1,38 @@
+<p style="text-align: center;">
+  <img src="/img/opera-ipfs-header.png" alt="IPFS in Opera Touch for iOS">
+</p>
+
+
+In 2020 we announced a big moment for IPFS: The first official support of IPFS protocol addressing in a major browser, when [Opera released IPFS support in their Android browser](ipns://blog.ipfs.io/2020-03-30-ipfs-in-opera-for-android/ "Opera Android IPFS announcement"). This was an important step in IPFS browser support generally, by building interest and momentum. We didn’t stop there, and we’ve got not just one… BUT TWO releases to share with you today:
```

Comment:
I'd love to use `ipns://` by default, but we are not there yet :-D 
```suggestion
In 2020 we announced a big moment for IPFS: The first official support of IPFS protocol addressing in a major browser, when [Opera released IPFS support in their Android browser](https://blog.ipfs.io/2020-03-30-ipfs-in-opera-for-android/). This was an important step in IPFS browser support generally, by building interest and momentum. We didn’t stop there, and we’ve got not just one… BUT TWO releases to share with you today:
```

Reply:
OMG this is what happens when u have IPFS support by default in a browser ;)